src/main/java/net/sf/jabref/gui/keyboard/KeyBindingsDialogView.java

```diff
@@ -0,0 +1,30 @@
+package net.sf.jabref.gui.keyboard;
+
+import net.sf.jabref.gui.FXAlert;
+import net.sf.jabref.logic.l10n.Localization;
+
+import com.airhacks.afterburner.views.FXMLView;
+import javafx.scene.control.DialogPane;
+import javafx.scene.control.Alert.AlertType;
+import javafx.stage.Stage;
+
+public class KeyBindingsDialogView extends FXMLView {
+
+    public KeyBindingsDialogView() {
+        super();
+        bundle = Localization.getMessages();
+    }
+
+    public void show(KeyBindingPreferences keyBindingPreferences) {
+        FXAlert keyBindingsDialog = new FXAlert(AlertType.INFORMATION, Localization.lang("Key bindings"));
+        keyBindingsDialog.setDialogPane((DialogPane) this.getView());
+        KeyBindingsDialogViewModel controller = (KeyBindingsDialogViewModel) fxmlLoader.getController();
+        controller.setKeyBindingPreferences(keyBindingPreferences);
+        controller.initializeView();
+        keyBindingsDialog.setResizable(true);
+        ((Stage) keyBindingsDialog.getDialogPane().getScene().getWindow()).setMinHeight(475);
```

Comment:
Well the minimum size of the dialog pane is already set to this dimension in the fxml file. However if you omit these lines you are able to resize the dialog window to whatever you like but not the dialog content. So all properties set in the fxml file only affect the dialog content but not the dialog window itself. With these lines omitted you could resize the dialog to a point where the content doesn't resize and it may look bugged.

![screen shot 2016-06-07 at 15 04 46](https://cloud.githubusercontent.com/assets/15340757/15858524/415e49b4-2cc1-11e6-84e9-ceaf7a1c5cde.png)